### PR TITLE
Fix the http port used not in sync with what is logged

### PR DESCRIPTION
**Pull request by graywolf336**
This was driving me bonkers as the port was different but it didn't display it in the logs 🙈

### CODE REVIEW

File: core/core.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func Start() error {
 	// start the rtmp server
 	go rtmp.Start(setStreamAsConnected, setBroadcaster)
 
-	port := config.WebServerPort
+	port := data.GetHTTPPortNumber()
```
Comment (gabek) [inline core/core.go:75]: This isn't correct if somebody overrides the port on the command line.  This is necessary if somebody is already running something on port 8080 and can't get to the admin to change it.

It should be getting set in https://github.com/owncast/owncast/blob/master/main.go#L112

Comment (graywolf336) [reply core/core.go:75]: Heh, you're right but there's a race condition which is causing the logging to be incorrect.

https://github.com/owncast/owncast/blob/1d9774e4533512ce708bdd7abeb10fa7e6a93806/main.go#L107-L109
Is called **before** the `config.WebServerPort` is set:
https://github.com/owncast/owncast/blob/1d9774e4533512ce708bdd7abeb10fa7e6a93806/main.go#L111-L122

Comment (gabek) [reply core/core.go]: ah yeah good catch!  `core.Start()` shouldn't be logging the web server port, that should be in `router.Start()`.

Comment (graywolf336) [reply core/core.go]: Logging moved to the appropriate package. :)

**Pull request merged**